content/en/network_monitoring/devices/network_topology_map.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Network Topology Map
+kind: documentation
+is_beta: true
+further_reading:
+- link: "/network_monitoring/devices/data"
+  tag: "Documentation"
+  text: "Data Collected with Network Device Monitoring"
+- link: "https://www.datadoghq.com/blog/monitor-snmp-with-datadog/"
+  tag: "Blog"
+  text: "Monitor SNMP with Datadog"
+---
+
+<div class="alert alert-warning">The Network Topology Map for Datadog Network Device Monitoring is in public beta.</div>
+
+## Overview
+
+The Network Topology map provides an overview of your network's physical connections, so you can more easily identify issues in your devices and understand their upstream and downstream impacts. 
+
+{{< img src="/network_device_monitoring/network_topology_map/network_topology_map_search.mp4" alt="The network topology map, with vendor:cisco added to the search bar, and then the Filter nodes box filtered by nyc. A node is selected and the inspect option chosen, showing its connected nodes. One of the connected nodes is then selected, and the inspect option is selected again, displaying additional connected nodes" video="true" >}}
+
+## Setup
+
+The Datadog Agent version 7.46 and later automatically collects topology data. No additional installation is necessary.
+
+### Prerequisites
+
+1. LLDP is enabled on the device with LLDP data exposed through SNMP.
+2. Datadog Agent version 7.46 or later is installed.
+
+## Investigating devices
+
+In addition to providing an overview of your network's physical connections, you can investigate individual devices to understand their connections, flows, and overall status. Hovering over a device displays its overall status and key metrics.
+
+{{< img src="/network_device_monitoring/network_topology_map/network_topology_map_device_hover.png" alt="The network topology map with the cursor hovering over a device" style="width:80%;" >}}
+
+ You can also click on a device to see the following options:
+
+{{< img src="/network_device_monitoring/network_topology_map/network_topology_map_device_options.png" alt="The network topology map with a device selected, displaying the options to Inspect, View device details, and view flow details" style="width:80%;" >}}
+
+### Inspect
+
+Choose **Inspect** to see the device's interface connections. Click on any of the connected interfaces for further investigation.
+
+{{< img src="/network_device_monitoring/network_topology_map/network_topology_map_device_inspect.png" alt="The Inspect view of an individual device, displaying the device's interface connections" style="width:80%;" >}}
```

Comment:
Optional, if you want to limit images on this page, I think the inspect and view device details are fairly self explanatory. I like the image that shows these menu options, but otherwise showing the details doesn't add that much value in my opinion.